landing-page/scripts/tasks_predict.py

```
@@ -83,14 +83,34 @@ def _run_medieval_inference(yolo_models, img_arr, image_bytes, mime_type, image_
     """
     stave_result = {}
     conn_holder: dict = {}
+    progress_lock = threading.Lock()
+    progress_state: dict = {"row": 0, "total": 0}
+
+    def _on_paco_progress(row: int, total: int) -> None:
+        with progress_lock:
+            progress_state["row"] = row
+            progress_state["total"] = total
 
     def _stave_pipeline():
```

Ruff (0.16.1)

[warning] Missing return type annotation for private function `_stave_pipeline`

Add return type annotation: `None`

(ANN202)